custom_containers/pytorch/mnist/Dockerfile

```diff
@@ -0,0 +1,49 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the \"License\");
+# you may not use this file except in compliance with the License.\n",
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an \"AS IS\" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Dockerfile
+FROM python:2.7.15-jessie
+WORKDIR /root
+
+# Installs pytorch and torchvision.
+RUN pip install torch torchvision
```

Comment:
nit: Do we need to specify specific versions for PyTorch & torchvision to guarantee this work?